package.json

```diff
@@ -142,10 +141,10 @@
     "ts-node": "^2.0.0",
     "tslint": "~4.3.1",
     "typedoc": "^0.5.3",
-    "typescript": "2.0.10",
+    "typescript": "~2.1.5",
```

Comment:
I'm running `TypeScirpt 2.1.5` in production on `Angular 2.4.3` and I have no issues with AOT compilation in either of my largest apps.

Perhaps it would be more accurate to say that isn't flakey. The language service issue was the initial blocker to TypeScript 2.1 which is referenced above.

I do on the other hand compile my AOT with `ngc ` directly which may support the `ngc-webpack` theory.